webpack.config.js

```diff
@@ -34,6 +33,8 @@ const vendorPath = resolve(staticPath, "vendor");
 function stylesheet(css) { return `<link href="{{ pathto('_static/${css}', 1) }}?digest=${this.hash}" rel="stylesheet" />`; }
 function preload(js) { return `<link rel="preload" as="script" href="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}" />`; }
 function script(js) { return `<script defer src="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}"></script>`; }
+// Adding FA without preloading
+function fascript(js) { return `<script src="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}"></script>`; }
```

Comment:
Could we rename these?

- script -> deferScript
- fascript -> script

Or combine into one function with a defer option?

- `function script(js, { defer = false } = {})`

Reply:
I have zero strong feelings re naming here so if you feel strongly I can make the change.